cypress/integration/AppAnalytics/DiseaseNoTrialsFound.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Disease listing page analytics
+
+    Scenario: Page Load Analytics fires when a user views the No Trials Found page for a disease
+        Given "trialListingPageType" is set to "Disease"
+        And "pageTitle" is set to "{{disease_label}} Clinical Trials"
+        And "analyticsPublishedDate" is set to "02/02/2011"
+        When the user navigates to "/C3037?cfg=0"
```

Comment:
Since `trialListingPageType` has been set to "Disease", I don't believe you need the query parameter "?cfg=0"

Reply:
Removed!